src/types/js-quantities/README.md

```diff
@@ -0,0 +1,16 @@
+# Installation

```

Comment:
Do we really need this file? Remove it and add the `js-quantiles` module must be installed by the user

Reply:
I'll remove it.